### ISSUE TITLE
Tell the user which profile was used for RockFrontend

### ISSUE THREAD

**Issue by webmanufaktur** [baumrock/RockFrontend#2]
It would be nice to see which profile was installed for RockFrontend.

![2022-08-10_21-45](https://user-images.githubusercontent.com/5909089/184006588-03411217-d70b-4d63-a2b6-4249dae012bf.png)

This is the settings page of RockFrontend after installation using the blank profile. Noone would ever know what might have been installed at some point.

**Comment by BernhardBaumrock** [comment]
Hey @webmanufaktur why do you think would that be helpful?
It's an easy thing, but I don't really see a benefit? It's just populating some files from the skeleton folders, but I don't see why seeing the used profile would be any better than just inspecting the folder structure from the IDE?

**Comment by webmanufaktur** [comment]
One benefit would be that even those without access to the source files could check up on this. 

Another could be in 6 months when I look into an "old" project I don't have to access any files, just login and see whats going on.

Or let's say I just test the module, something doesn't seem right and I couldn't even check I really did what I thought I did. Imagine I chose the UIKIT profile but there wouldn't be anything. I couldn't even tell I installed the profile. Or I already installed one and install another just because I thought I missed a step.

Last but not least it confused me looking at that screen even though I thought I already installed something. See it as a success message like: You already installed [THIS] profile the setup is done.

It's just better user experience in my opinion.
